src/main/java/it/gov/pagopa/pu/debtpositions/citizen/Constants.java

```diff
@@ -0,0 +1,9 @@
+package it.gov.pagopa.pu.debtpositions.citizen;
+
+public class Constants {
+
+    public enum PERSONAL_DATA_TYPE {
```

Comment:
I don't like Constants class inside which define enum... let's define the enum as a new Class instead